packages/web/src/scss/components/Skeleton/_tools.scss

```
@@ -0,0 +1,38 @@
+@use 'sass:map';
+@use '@tokens' as tokens;
+@use '../../tools/breakpoint';
+@use '../../tools/string' as spirit-string;
+@use '../../tools/typography';
```

```suggestion
@use '../../tools/string' as spirit-string;
```

sorry I missed this before